Tasks/choco/Choco.ps1

```diff
@@ -5,17 +5,17 @@ param(
 
     [Parameter()]
     [string] $Version,
-
+ 
     [Parameter()]
-    [string] $IgnoreChecksums
+    [string] $ignoreChecksums
 )
 
+Set-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1
```

Comment:
This looks like something that could potentially have other side effects on the system and wouldn't be intuitive that this script is setting it. What problem is this helping solve in this script?

Reply:
Yes, I think we should remove it. If we want to do this, it probably belongs in a more generic task than the chocolatey task.

Reply:
I made a request to Ruben to have long file paths enabled by default in the OS images instead of doing it in our tasks, so please remove this.

Reply:
David should we also have CPC make that change for their OS images? Or set it during provisioning?

Reply:
I can make a task in this repo that enables long file paths, and users can choose to apply it as part of their customizations. I don't know if CPC will want to enable long file paths for images that are used by non-developers, although in the year 2023 I can't think of a good reason why Windows doesn't ship with this enabled by default.